pkg/hostpath/hostpath.go

```diff
@@ -347,6 +363,28 @@ func shareUpdateRanger(key, value interface{}) bool {
 			klog.Warningf("share %s vol %s target path %s delete error %s",
 				shareId, volID, oldTargetPath, err.Error())
 		}
+		//TODO removing contents from a read only volume, where we employ an intermediate bind mount, is the single
+		// item in our list of update content features that still works when this driver is restarted after a Pod
+		// is started with one of our volumes.  The question is do we even bother supporting this, or just make
+		// the general statement that "results may vary" and do not claim any production level support for updating
+		// contents with read only volumes since we don't have a comprehensive solution for when the driver is restarted.
+		if hpv.IsReadOnly() {
```

Comment:
To elaborate on the TODO comment above

I was able to verify this multiple times in my testing. However, why I am able to remove content under the kubelet mount dir after a restart, but am unable to update or change content under the kubelet mount dir after a restart, is still a mystery to me.

Pending learning the precise why, I also wonder if this limited form of update support for read only volumes after driver restart will be worth the effort to maintain, document, explain, etc. moving forward will be worth it.

Reply:
I'm fine leaving this for now - Tech Preview means we have freedom to explore these things and figure out what we can ultimately support.

Reply:
sounds good thanks